py/kubeflow/testing/ci/kf_unittests.py

```diff
@@ -0,0 +1,14 @@
+"""Uses an Argo example to generate to create a workflow"""
```

Comment:
nit:

"""A smoke test to ensure run_e2e_workflow.py can properly trigger workflows using a py_func.

This module defines a simple py_func which will return the Argo workflow resulting from loading the specified
YAML file.

The purpose of this is to allow the presubmits/postsubmits to verify that run_e2e_workflow.py is actually
triggering workflows using py_func.
"""

Reply:
Added.